client/src/app/barmen/new/barman-new.component.ts

```diff
@@ -40,7 +40,8 @@ export class BarmanNewComponent implements OnInit {
             lastName: new FormControl('', [Validators.required]),
             firstName: new FormControl('', [Validators.required]),
             nickname: new FormControl('', [Validators.required]),
-            facebook: new FormControl(''),
+            facebook: new FormControl('',
+                [Validators.pattern(/^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/)]),
```

Comment:
Same here

Reply:
Because I was lazy and I found this regex :) Be free to re work on it !

Reply:
Ok I will do that